packages/manager/apps/pci-instances/package.json

```diff
@@ -16,12 +16,12 @@
     "type:check": "tsc --noEmit"
   },
   "dependencies": {
-    "@ovh-ux/manager-config": "^7.3.2",
-    "@ovh-ux/manager-core-api": "^0.8.0",
-    "@ovh-ux/manager-pci-common": "^0.7.2",
-    "@ovh-ux/manager-react-components": "^1.26.0",
-    "@ovh-ux/manager-react-core-application": "^0.10.0",
-    "@ovh-ux/manager-react-shell-client": "^0.7.0",
+    "@ovh-ux/manager-config": "^7.5.3-alpha.0",
```

Comment:
@lionel95200x 
I heard that the Core team has mismached the versions, it will be bumped by their side very soon.
But I need this packages for now ...